_component_design/actions-menu.md

```diff
@@ -1,17 +1,103 @@
 ---
 title: Actions Menu
-parent: Editing
-secondary: Inputs and Controls
+parent: Inputs and Controls
+secondary: Editing
 layout: component
 category: Components
 usage: >
   The actions menu is used anytime a group of actions can be performed on a specific object on the current page or the current page. By grouping these actions within the actions menu, it allows users to have a single location to trigger multiple types of actions.
-preview-image: components/preview-help.svg
-status: unknown
-resource: false
-last-modified: 2017-08-17
+preview-image: preview-images/actions-menu.svg
+status: stable
+resource: true
+last-modified: 2017-11-03
 ---
 
 {% include toc.html %}
 
-<!-- unknown -->
+## Introduction
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+
+- An actions-menu is used to consolidate multiple related actions, displayed as links, into a single popover menu. 
+
+### When not to use
+
+- When a table object has a limited number of actions that can be taken upon them it is best to surface these options by another means. For example if a table object can only be deleted or created it is best to surface these via table controls at the top of the table. For additional guidance see [tables]({{site.baseurl}}/components/tables.html) documentation.
```

Comment:
If a table object has only a limited number of associated actions (such as create and delete), use table controls at the top of the table. For more information, see [tables]({{site.baseurl}}/components/tables.html).

Reply:
updated.